def f1():
    name = 'hard'
    try:
        a = 3
    except Exception as e:
        pass

    print('a')

def f2():
    print('b')
